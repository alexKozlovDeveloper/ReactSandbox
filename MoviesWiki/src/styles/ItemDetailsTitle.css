.title {
    font-size: 26px;
    margin: 2px;
    padding: 2px;
}