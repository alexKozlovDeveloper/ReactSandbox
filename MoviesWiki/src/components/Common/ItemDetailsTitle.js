import React, { Component } from "react";

import styles from "../../styles/ItemDetailsTitle.css"

function ItemDetailsTitle(props) {
    return  <div className={styles.title}>
                {props.title}
            </div>;
}

export default ItemDetailsTitle;