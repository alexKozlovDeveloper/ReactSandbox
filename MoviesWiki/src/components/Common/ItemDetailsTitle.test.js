import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import ItemDetailsTitle from "./ItemDetailsTitle";

configure({adapter: new Adapter()});

test('ItemDetailsTitle rendering test', () => {

    var title = "someTitle";

    const output = shallow(<ItemDetailsTitle title={title} />);
    expect(shallowToJson(output)).toMatchSnapshot();
 });

