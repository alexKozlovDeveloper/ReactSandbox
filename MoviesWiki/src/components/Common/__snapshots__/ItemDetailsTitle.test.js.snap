// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemDetailsTitle rendering test 1`] = `
<div
  className="title"
>
  someTitle
</div>
`;
